### PR TITLE
fix(doc): URL deprecated

### PR DESCRIPTION
**Pull request by kozfelipe**
url deprecated

### CODE REVIEW

File: doc/en/developer/writemodule.rst [hunk 1]
```diff
@@ -61,8 +58,7 @@ inside the *conf.php* file.
 This SQL file is executed during the module installation if it is
 configured inside the *conf.php* file. If you want your module to be
 available from Centreon menus, you must insert new entries into the
-``topology`` table of the ``centreon`` database. An example is
-available inside the ``Dummy`` module.
```
Comment (kozfelipe) [inline doc/en/developer/writemodule.rst]: no such module

**Comment by kozfelipe** [comment]
https://github.com/centreon/centreon-dummy/issues/1

**Comment by lpinsivy** [comment]
@lpinsivy readd dummy on github

**Comment by ganoze** [comment]
Closing in favor of #7912

**Pull request closed without merging**